userspace/libsinsp/test/events_proc.ut.cpp

```diff
@@ -319,19 +319,20 @@ TEST_F(sinsp_with_test_input, spawn_process)
 	sinsp_evt* evt = NULL;
 
 	uint64_t parent_pid = 1, parent_tid = 1, child_pid = 20, child_tid = 20;
+	uint64_t exe_ino = 242048, ctime = 1676262698000004577, mtime = 1676262698000004588;
 	scap_const_sized_buffer empty_bytebuf = {.buf = nullptr, .size = 0};
 
 	add_event_advance_ts(increasing_ts(), parent_tid, PPME_SYSCALL_CLONE_20_E, 0);
 	std::vector<std::string> cgroups = {"cpuset=/", "cpu=/user.slice", "cpuacct=/user.slice", "io=/user.slice", "memory=/user.slice/user-1000.slice/session-1.scope", "devices=/user.slice", "freezer=/", "net_cls=/", "perf_event=/", "net_prio=/", "hugetlb=/", "pids=/user.slice/user-1000.slice/session-1.scope", "rdma=/", "misc=/"};
 	std::string cgroupsv = test_utils::to_null_delimited(cgroups);
 	std::vector<std::string> env = {"SHELL=/bin/bash", "PWD=/home/user", "HOME=/home/user"};
 	std::string envv = test_utils::to_null_delimited(env);
-	std::vector<std::string> args = {"--help"};
+	std::vector<std::string> args = {"-c", "'echo aGVsbG8K | base64 -d'"};
```

Comment:
is that change done to test multiple arguments to the binary?

Reply:
yes